src/components/About/About.jsx

```diff
@@ -44,16 +44,13 @@ const About = () => (
                 Other Relevant Links
             </StaticH1>
             <StaticP>
-                Look at the following links to get a feel of our culture, the organization structure, event vibes, how the different Core Teams work, how to get involved, access to free resources, and more.
+                Look at the following links to get a feel of our culture, our organizational structure, event vibes, how our teams work, how to get involved, access to free resources, and more.
                 <EventList>
                     <li><a href="https://docs.google.com/presentation/d/122LR6C2yGpHYKe4mKHWFF98y6F8sV9Jh0dLnogiKnug/edit?usp=sharing">Our Mass Meeting Slides</a></li>
-                    <li><a href="https://docs.google.com/presentation/d/1ctHSblSXA9FiPauXlbR8ypy6dxF_zp4tH2ICU-_HKN0/edit">The Core Teams</a></li>
-                    <li><a href="https://calendar.google.com/calendar/selfsched?sstoken=UVA3dk1tZkxNOFpCfGRlZmF1bHR8ODMwMzEwNjE4YzY3Y2I0OThlZGIzY2IwMGU1MDUwMGI">Office hours signup</a></li>
+                    <li><a href="https://docs.google.com/presentation/d/1cihP20nwMc4rEwrapCwK8tumh5eUCUIdshGuB9-QoPY/edit">Our Teams</a></li>
+                    <li><a href="https://calendar.google.com/calendar/u/0/selfsched?sstoken=UUZvRjlEM0RMUlZ2fGRlZmF1bHR8NjkzODc3M2I3MDJlZGM5NDQyZDU2M2IzMzc4YTM2NjY">Office hours sign-up</a></li>
```

Comment:
I think the `/u/0` should not be hardcoded into this link? Because if people have multiple Google accounts signed in and their umich email isn't index 0 it might force them to schedule using a different account.

Reply:
Good catch, I'll fix that